pages/Network_APIs/WorkflowforContribution.md

```diff
@@ -0,0 +1,69 @@
+We welcome and encourage contributions from the broader community. If you are interested in becoming a member of the 5G-MAG and actively participating in shaping this work, please contact the [Project Office](https://www.5g-mag.com/contact)
+
+# Required interactions to exploit network capabilities
+  
+## Pre-conditions
+  * The production company has set up an agreement for Network Capability usages. As result, the production company obtained information, used for authentication. During the process of negotiating the agreement, a set of relevant network capabilities has been identified (e.g. using an API catalogue). As result, the production company is authorized to use certain network capabilities (when available).
+  * The production crew has access to one, or several, Network API Platforms. This platform is accessible through any device/connectivity (e.g. a website accessible through the open internet, or command prompt, a dedicated app, etc …).
+    * Note: For the Network API Platform access, the crew has obtained key access tokens/key/credentials/payment-details in advance.
+  * The production crew have a set of credentials (SIM/eSIM) of the network devices will connect to. The network provides, by default, best-effort connectivity.
+ 
+## Before the Event: 
+ 
+### Step 1) Preparing your device, configuring application clients & servers and configuring client/server flows
+  * Devices have already best effort connectivity .
+  * Through app specific [AppAPI], the client app is properly connected to the Application Server. e.g. Haivision app with BBC server, or LiveU app with LiveU server, etc …
+ 
+### Step 2)Event planning and pre-booking
+  * a) Through the Network API Platform, the production crew can discover the capabilities the network can offer (for which the production company is eligible for).
+    * Example: QoD available, connectivity monitoring available, Timing as a service available, edge compute instantiation, …
+  * b) Through the Network API Platform, the production crew requests network services for every of the planned SIM cards in advance. Possible services (Network Capabilities) are:
+    * Quality on Demand
+      * One or several QoS profiles for each SIM card (QOS Profiles are mapped to 5QIs) 
+      * Example: A sim may be pre booked for One uplink video / One uplink audio / one downlink data / …
+    * Time as a service
+      * Access stratum based or PTP based
+    * Note: Booking is done based on :
+      * Geographical area
+      * Schedule (starting time and closing time of the event)
+  * c) Through the Network API Platform, the production receives a booking offer responding to the service request.
+  * d) Through the Network API Platform the production accepts the service booking offer (involving payment/contract/SLA aspects)[TL6] .
+  * e) Through the production receives connection IDs to be used for every reserved SIM/services
```

Comment:
Should e) be "Through the production receives connection IDs to be used for every reserved SIM/services"
Is a "connection ID" a parameter that identifies the SIM with the booked services?